.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
+on:
+- pull_request
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: thomasjpfan/labeler@master
```

Comment:
question, does this have to be on your user?

Reply:
From a comment above
> I forked the labeler repo and added a max-labels feature

It's probably safer than running third party code from master (or a tag which can be force pushed) anyway.

Reply:
I just don't know what this thing does really, trying to understand :D

Reply:
I had to update this PR with my `v2.2.0` tag.